### DISCUSSION TITLE
addressOverrides as environment variable

### DISCUSSION THREAD

**Discussion by ghost** [hyperledger/fabric#2551] [Q&A]
Is there any way to give addressOverrides to peer as environment variables? I haven't found any documentation on this and this is currently causing quite a bit of trouble with our network.

**Comment by C0rWin** [comment]
To override via environment variables, you should do `export CORE_PEER_DELIVERYCLIENT_ADDRESSOVERRIDES=`. Basically, you can follow the pattern where config properties always prefixed with `CORE` and then traverse the config path where each level split by `_` (underscore).

**Reply by ghost** [reply]
But how does the structure work when addressoverrides can include multiple overrides and each override can include to, from and cacertsfile variables?
